### PR TITLE
[Localization] MSBuild - WIP

### PR DESCRIPTION
**Pull request by wachaudh**
Localization of MSBuild.

### CODE REVIEW

File: msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CoreMLCompilerTaskBase.cs [hunk 1]
```diff
@@ -224,10 +225,10 @@ public override bool Execute ()
 						if ((Compile (model, outputDir, log, partialPlist)) != 0)
 							return false;
 					} else {
-						Log.LogMessage (MessageImportance.Low, "Skipping `{0}' as the output file, `{1}', is newer.", model.ItemSpec, partialPlist);
+						Log.LogMessage (MessageImportance.Low, MSBStrings.M0119, model.ItemSpec, partialPlist);
 					}
 				}
-
+
```
Comment (wachaudh) [inline msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CoreMLCompilerTaskBase.cs]:
```suggestion

```

File: msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs [hunk 2]
```diff
@@ -31,12 +32,12 @@ public override bool Execute ()
...
 			// LinkWith must be migrated for NoBindingEmbedding styled binding projects
 			if (NativeReferences.Length == 0) {
-				Log.LogError (7068, null, $"Can't create a binding resource package unless there are native references in the binding project.");
+				Log.LogError (7068, null, MSBStrings.E7068);
 				return false;
 			}
 
 			string bindingResourcePath = Path.Combine (ProjectDir, OutputPath, Path.ChangeExtension (Path.GetFileName (BindingAssembly), ".resources"));
-			Log.LogMessage ($"Creating binding resource package: {bindingResourcePath}");
+			Log.LogMessage (String.Format (MSBStrings.M0121, bindingResourcePath));
```
Comment (spouliot) [inline msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs]: why does this one needs `String.Format` while the others do not ?
seems a different overload is used

Comment (wachaudh) [reply msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs]: It doesn't, it's a mistake. Fixed.

**Comment by wachaudh** [comment]
build

**Pull request closed without merging**